### DISCUSSION TITLE
Compartment temperature in adiabatic case higher with leakage venting vs no venting

### DISCUSSION THREAD

**Discussion by SFC-CA** [firemodels/cfast#2063] [Q&A] [answered]
Hi all,

I have some results I can't explain. More than happy for it to be a conceptual misunderstanding on my part. I came across it with a different model but get the same behaviour with the simplified model discussed below:

My base model has one compartment 50m x 50m x 10m (WLH) with a constant fire output of 5MW running for 1 hour. The difference between the two is that the first model has no leakage at all, where as the the second has leakage at a rate of 0.0012m2/m2 over the walls. There are no material properties set so the model should be running as adiabatic. I have also tried this with ticking the adiabatic option and get the same results. 

The model without leakage reaches an UL peak of 845oC and the model with leakage gets to 1,180oC. The LL temps are similar. I expected the leakage model to have a lower temperature. Tracking enthalpy, the no leakage model matches with what I would expect, but obviously the leakage model does not. See attached png. I would expect enthalpy exchange through the vent with net outflow so the average temperature in the compartment should go down. The actual HRR is as expected and the compartment size has enough oxygen above the limit to sustain the fire for at least 1 hour. So, the no vent case is not underventilated and introduction of a vent does not allow more energy release in the compartment in this case.

![Enthalpy](https://github.com/firemodels/cfast/assets/52268149/b4e0689d-0938-4b97-affb-85253a40565e)

If my working out is right, the temperature in the leakage case defies energy conservation. Again, more than happy to be shown the error of my ways here. 

Note that I have also tried running the model manually inputting the wall vent rather than relying on the leakage input (sill at 5%, total height at 90% of compartment height). The results were the same.  

So, what am I missing? Thanks in advance. 

Input files are below:

---------------------------------------------------------------------------------------------------------------------------------------------------

&HEAD VERSION = 7700, TITLE = 'TA1_NoVenting' /
 
!! Scenario Configuration 
&TIME SIMULATION = 3600 PRINT = 60 SMOKEVIEW = 15 SPREADSHEET = 15 / 
&INIT PRESSURE = 101325 RELATIVE_HUMIDITY = 50 INTERIOR_TEMPERATURE = 20 EXTERIOR_TEMPERATURE = 20 /
 
!! Compartments 
&COMP ID = 'Comp 1'
      DEPTH = 50 HEIGHT = 10 WIDTH = 50
      ORIGIN = 0, 0, 0 GRID = 50, 50, 50 /
 
!! Fires 
&FIRE ID = 'New Fire 1'  COMP_ID = 'Comp 1', FIRE_ID = 'New Fire 1'  LOCATION = 25, 10 / 
&CHEM ID = 'New Fire 1' CARBON = 1 CHLORINE = 0 HYDROGEN = 4 NITROGEN = 0 OXYGEN = 0 HEAT_OF_COMBUSTION = 50000 RADIATIVE_FRACTION = 0.35 / 
&TABL ID = 'New Fire 1' LABELS = 'TIME', 'HRR' , 'HEIGHT' , 'AREA' , 'CO_YIELD' , 'SOOT_YIELD' , 'HCN_YIELD' , 'HCL_YIELD' , 'TRACE_YIELD'  /
&TABL ID = 'New Fire 1', DATA = 0, 5000, 0, 0.09, 0, 0, 0, 0, 0 /
&TABL ID = 'New Fire 1', DATA = 3600, 5000, 0, 0.09, 0, 0, 0, 0, 0 /
 
&TAIL /

---------------------------------------------------------------------------------------------------------------------------------------------------

&HEAD VERSION = 7700, TITLE = 'TA2_venting' /
 
!! Scenario Configuration 
&TIME SIMULATION = 3600 PRINT = 60 SMOKEVIEW = 15 SPREADSHEET = 15 / 
&INIT PRESSURE = 101325 RELATIVE_HUMIDITY = 50 INTERIOR_TEMPERATURE = 20 EXTERIOR_TEMPERATURE = 20 /
 
!! Compartments 
&COMP ID = 'Comp 1'
      DEPTH = 50 HEIGHT = 10 WIDTH = 50
      ORIGIN = 0, 0, 0 GRID = 50, 50, 50 LEAK_AREA_RATIO = 0.0012, 0 /
 
!! Fires 
&FIRE ID = 'New Fire 1'  COMP_ID = 'Comp 1', FIRE_ID = 'New Fire 1'  LOCATION = 25, 10 / 
&CHEM ID = 'New Fire 1' CARBON = 1 CHLORINE = 0 HYDROGEN = 4 NITROGEN = 0 OXYGEN = 0 HEAT_OF_COMBUSTION = 50000 RADIATIVE_FRACTION = 0.35 / 
&TABL ID = 'New Fire 1' LABELS = 'TIME', 'HRR' , 'HEIGHT' , 'AREA' , 'CO_YIELD' , 'SOOT_YIELD' , 'HCN_YIELD' , 'HCL_YIELD' , 'TRACE_YIELD'  /
&TABL ID = 'New Fire 1', DATA = 0, 5000, 0, 0.09, 0, 0, 0, 0, 0 /
&TABL ID = 'New Fire 1', DATA = 3600, 5000, 0, 0.09, 0, 0, 0, 0, 0 /
 
&TAIL /

**Comment by drjfloyd** [comment]
Leakage is treated as a narrow vent. Look at the flow and hrr outputs. You
will eventually have bi-directional flow in the leak path and the hrr that
results will far outweigh the cooling effects of the air exchange.

**Reply by SFC-CA** [reply]
Thanks, Jason.
I don't doubt there is bi-directional flow, but that the temperature increase in the compartment seems to exceed that possible by the energy released in the compartment. In the no leakage case, the HRR is not limited, so incoming oxygen from outside shouldn't have an effect in this case. The no leakage case should present a maximum, no?

**Comment by drjfloyd** [comment, marked as answer]
The internal energy of the sealed compartent is dP V_0 + rho_0 V c dT.  Your pressure in the sealed compartment is rising by 290 kPa. Alot of the fire energy (about 40 %) is going into raising the pressure.  Ignoring the energy loss from the vent outflow, you would expect 60 % of the temperature rise in the sealed case vs. the open. The actual value is 70 % but loosing 10 % of the energy from the leakage flow doesn't seem unreasonable.

**Reply by SFC-CA** [reply]
Again, thanks for the reply, and sorry to take up your time with a physics lesson. Clearly I neglected pressure. Look, I think I understand what you are saying but it seems non-intuitive to me. I need to have a think about it. 

In the mean time, I now have a new problem, and that is reconciling the energy in the sealed case.

Using the equation you gave above (which is for change in enthalpy, right?) I wanted to look at the energy balance in the sealed case:

The compartment volume is (50x50x10m)=25e3 m3
The pressure increase is 290e3 Pa
rho is 1.2 kg/m3
c is 1012/1.4 = 723 J/kgK (this is constant volume, right?)
and dT is 845-20 = 825 K

dH = dP V_0 + rho_0 V c dT

So dH= 290e3 x 25e3 + 1.2 x 25e3 x 723 x 825 = 25.1e9 J
But energy from the fire = 5e6 W x 3600s = 18e9 J
Although if I just look at temperature change alone:
1.2 x 25e3 x 723 x 825 = 17.9e9 J which is pretty much the energy from the fire. 

I know you guys have a lot on your plate so I appreciate the time you've taken here. Can you please tell me where I've gone wrong? 

Thanks!

**Comment by drjfloyd** [comment]
The quantity you are conserving in sealed volume is internal energy (U):

U = rho_0 V_0 c_p dT - dP V_0 = rho_0 V_0 c_v dT

when you use constant volume specific heat you are accounting for pressure with that.

**Comment by SFC-CA** [comment]
OK, that's what I thought. I still need to convince myself of the sealed vs non-sealed case, though.

Thanks for your help, Jason. Much appreciated.
